docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -12,7 +12,7 @@ We then walk you through building your first {project-full-name} application, di
 == Introducing Spring Cloud Sleuth
 
 Spring Cloud Sleuth provides API for distributed tracing solution for https://cloud.spring.io[Spring Cloud].
-It integrates out of the box with two tracer implementations:
+It integrates out of the box with tracer implementation:
```

Comment:
Because we now have a list with only one item, we can simplify this line and the list, as follows:

It integrates with https://github.com/openzipkin/brave[OpenZipkin Brave].

Reply:
Done